README.md

```diff
@@ -96,17 +170,55 @@ Yes! *She's Coding* is build by the community, for the community. We currently h
 
     See [https://help.ubuntu.com/community/PostgreSQL]
 
-    For simple setup (local use only) follow the "Alternate Server Setup" of the
-resource provided above.
+    For simple setup (local use only) follow the "Alternate Server Setup" of the resource provided above.
 
-+ Set up the database:
-  ```
-  $ createdb shescoding_development
-  $ psql -d shescoding_development -c "CREATE USER shescoding WITH PASSWORD 'shescoding'"
-  $ psql -d shescoding_development -c "GRANT ALL PRIVILEGES ON DATABASE shescoding_development to shescoding"
-  ```
+  + Windows 10:
+
+      (NOTE: Google Chrome will block these downloads for security reasons. Please download using Firefox or Edge)
+
+      Install postgres from [EnterpriseDB](https://www.enterprisedb.com/downloads/postgres-postgresql-downloads)
+
+      Set up your username and password defaults. You will not need to set up a specialized name or password when developing in Windows
+
+      Install PGAdmin4 with postgres in the installer, or download manually from [PGAdmin for Windows](https://www.pgadmin.org/download/pgadmin-4-windows/)
+
+
+---
+
++ Set up the database
+
+  + Mac or Ubuntu
```

Comment:
may be Linux? I believe those instructions suppose to work regardless of what os linux machine has installed (ubuntu, Fedora, opensuse, etc.)